### DISCUSSION TITLE
Hw4 Part 3 question 2

### DISCUSSION THREAD

**Discussion by ElisDiaz46** [cooperrc/computational-mechanics#180] [Q&A]
Hello,

How come I am having this error for this code?

![image](https://user-images.githubusercontent.com/116235922/212398987-a684d654-dbd9-465d-a5e2-280d8da1df65.png)

![image](https://user-images.githubusercontent.com/116235922/212399215-f90f5f06-015f-4bf9-bb8e-73f7dd134c38.png)

**Comment by Maid-Unknown** [comment]
If you print out the sin_data, you will see it is str type. You should separate the str to float

**Reply by cooperrc** [reply]
great idea, I think you'll see that the `sin_data` values all have ` '...' ` quotes to show they are strings

**Comment by cooperrc** [comment]
add a `skiprows = 1` argument so you're not trying to turn the headings into numbers, 

```python 
np.loadtxt('../data/stress_relax.dat', skiprows = 1, delimiter = ',')
```
